lighthouse-cli/performance-experiment/server.js

```diff
@@ -0,0 +1,106 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const http = require('http');
+const fs = require('fs');
+const parse = require('url').parse;
+const path = require('path');
+
+const ROOT = `${__dirname}/src`;
+const FOLDERS = {
+  REPORTS: `${ROOT}/reports`
+};
+
+const server = http.createServer(requestHandler);
+server.on('error', e => console.error(e.code, e));
```

Comment:
should use [`log.error`](https://github.com/GoogleChrome/lighthouse/blob/8142268f8323b1b7dd0be34a7d0d345a9aed99a6/lighthouse-core/lib/log.js) here for logging